v22.1/create-and-configure-changefeeds.md

```diff
@@ -63,11 +63,18 @@ To create an {{ site.data.products.enterprise }} changefeed:
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> CREATE CHANGEFEED FOR TABLE table_name, table_name2 INTO '{scheme}://{host}:{port}?{query_parameters}';
+CREATE CHANGEFEED FOR TABLE table_name, table_name2 INTO '{scheme}://{host}:{port}?{query_parameters}';
 ~~~
 
 {% include {{ page.version.version }}/cdc/url-encoding.md %}
 
+If you do not specify a sink or the `format` option, then the changefeed will hang until canceled. If you create a changefeed without a sink but specify the [`format=csv` option](create-changefeed.html#format), your changefeed will run as a [core-style changefeed](changefeed-for.html) sending messages to the SQL session. For example:
```

Comment:
This isn't quite how this worked. The `format` option that affects behavior for a sinkless changefeed isn't the changefeed `format` option, but a command-line option for our client that affects how all data is returned. To make it return in csv format, you needed to run `cockroach sql --format=csv`.

The "hanging" is client-side behavior, not anything different about what Cockroach is doing. With the default format for an interactive SQL client, CockroachDB is sending the client the core changefeed events, but the client is buffering forever waiting for the query to finish (which it never will for most changefeeds) because the default format needs to know the maximum row length. Most formats won't do this--for example, you can also do `--format=ndjson`.